app/views/includes/scripts.njk

```diff
@@ -1,7 +1,9 @@
-<!-- Javascript -->
+{# Javascript #}
 <script type="text/javascript">
   var i18n = {{ translationStrings | safe }};
 </script>
+{# Promise polyfill for IE - https://github.com/taylorhakes/promise-polyfill#cdn-polyfill-use #}
+<script src="https://cdn.jsdelivr.net/npm/promise-polyfill@8/dist/polyfill.min.js"></script>
```

Comment:
I think it's better to download this dependency and host it ourselves in `assets/` directory.
Also, as discussed with @brid we can add `integrity` parameter to check if the file was altered somehow.